imports/plugins/included/surcharges/server/no-meteor/util/helpers.js

```diff
@@ -0,0 +1,23 @@
+// TODO: EK - merge this into a central location
```

Comment:
Looks like you have a note about merging these, but they still exist in two different files.

Reply:
Yes... I was wondering thoughts on where we should put something like this? We use it three different places, do we have some sort of `lib` folder we use / should I create one for something like this?